application/views/ride_views.py

```diff
@@ -0,0 +1,38 @@
+from flask_restplus import Resource, Namespace, fields
+from flask import request, jsonify
+from datetime import datetime
+
+
+api = Namespace('Ride offers', Description='Operations on Rides')
+
+# data structure to store ride offers
+
+rides = {1: {
+    "destination": "Nairobi",
+    "route": "Thika",
+    "start_point": "Thika",
+    "start_time": "June 01 1990 09:00AM",
+    "requests": [],
+    "available_space": 10}
+}
```

Comment:
Please explain why this rides data is hardcoded here.

Reply:
It's just a data seed for the program